RELEASE_PROCESS.md

```diff
@@ -24,7 +24,7 @@
   [Look for an example here.](https://github.com/openshift/release/pull/3623). Wait that it gets merged. Make sure you have all the files in there which is two in `ci-operator/config` and one in `ci-operator/job`. Here is a handy script where you just need to change the version manually :
 
   ```bash
-   % for i in **/*tektoncd*0.3.0*;do sed -e "s/0\\\.3\\\.0/0\\\.4\\\.0/g" -e "s/0.3.0/0.4.0/" $i > ${i/0.3.0/0.4.0};done
+   % for i in $(find . -name '*tektoncd*0.3.0*');do sed -e "s/0\\\.3\\\.0/0\\\.5\\\.2/g" -e "s/0.3.0/0.5.2/" $i > ${i/0.3.0/0.5.2};done
```

Comment:
i think it may be best if you base from a more recent version (ie: 0.3.0 to something newer), you can mention in the documentation to use the latest version as well?

Reply:
Done.